.github/pull_request_template.md

```diff
@@ -20,16 +20,14 @@
 
 ## Checklists
 
-### Environment variables or secrets
+<!-- Delete if changes DO include secrets -->
+- [ ] No secrets were added
```

Comment:
I'm not sure we need this one - if secrets are inadvertently exposed they should be detected by `detect-secrets` which runs on pre-commit